src/libsumo/libsumo.i

```diff
@@ -297,6 +297,7 @@ def simulationStep(step=0):
 %template(TraCIConnectionVector) std::vector<libsumo::TraCIConnection>;
 %template(TraCILogicVector) std::vector<libsumo::TraCILogic>;
 %template(TraCIStageVector) std::vector<libsumo::TraCIStage>;
+%include "traci-server/TraCIConstants.h"
```

Comment:
libsumo should not depend on traci-server but maybe we should move the constants to libsumo anyway?

Reply:
Moving the constants to libsumo sound reasonable to me. Shall I include it in this pull request?

Reply:
Yes.